### PR TITLE
Fix: range outbound bug

### PR DESCRIPTION
**Pull request by lexich**
Fix issue #571

### CODE REVIEW

**Comment by TrySound** [comment]
@lexich Please, add tests where it fails

**Comment by lexich** [comment]
@TrySound I add test case. Css isn't very valid, but bug is bug.

**Comment by TrySound** [comment]
@lexich Squash commits please

**Comment by lexich** [comment]
@TrySound OK

**Comment by ai** [comment]
Release in 6.1.2

File: test/cases/gradient-fix.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.test {
+  background-image: -webkit-linear-gradient(yellow 50%, red 50%);
+  background-image: linear-gradient(to   , red 50%);
```
Comment (yisibl) [inline test/cases/gradient-fix.css]: Why use the wrong syntax?

> linear-gradient(to   , red 50%)

**Comment by lexich** [comment]
@yisibl It's a generated code from http://bourbon.io/ and code crush in this case.

**Pull request merged**